resources/services/lambda/functions.go

```diff
@@ -1272,3 +1416,12 @@ func resolveLambdaFunctionEventSourceMappingAccessConfigurations(ctx context.Con
 	}
 	return resource.Set(c.Name, data)
 }
+
+// ====================================================================================================================
+//                                                  User Defined Helpers
+// ====================================================================================================================
+
+type FunctionWrapper struct {
+	*lambda.GetFunctionOutput
+	UrlConfig *lambda.GetFunctionUrlConfigOutput
+}
```

Comment:
I don't think it is a 1-1 relationship between function and URLConfig. I think it is a 1-1 relationship between Function Alias and URL Config

Reply:
Thanks. I will fix it